### DISCUSSION TITLE
Can i join Team

### DISCUSSION THREAD

**Discussion by Nishant9947** [ROM-builders/temporary#10609] [General]
EvoX :- https://cirrus-ci.com/build/4762709563539456
AEX :- https://cirrus-ci.com/build/4560038009241600

DT :- https://github.com/Shivam1143/android_device_asus_X01AD/tree/EvoXX

**Comment by Apon77** [comment]
> EvoX :- https://cirrus-ci.com/build/4762709563539456 AEX :- https://cirrus-ci.com/build/4560038009241600
> 
> DT :- https://github.com/Shivam1143/android_device_asus_X01AD/tree/EvoXX

check #learn in telegram group

**Reply by Nishant9947** [reply]
Ok bruh 😁☺️
